TransbankPosSDK/Utils/TransbankWrap.cs

```diff
@@ -58,6 +58,11 @@ public class TransbankWrap {
     return ret;
   }
 
+  public static string last_sale() {
```

Comment:
This changes are related to las_sale()

Reply:
Autogen :(

Reply:
Yeah, but because you merged the las_sale wrapper gen with the get_totals.